pkg/plugin/api/client.go

```diff
@@ -100,25 +100,25 @@ func (c *Client) List(ctx context.Context, key store.Key) (*unstructured.Unstruc
 }
 
 // Get retrieves an object from the dashboard's objectStore.
-func (c *Client) Get(ctx context.Context, key store.Key) (*unstructured.Unstructured, bool, error) {
+func (c *Client) Get(ctx context.Context, key store.Key) (*unstructured.Unstructured, error) {
```

Comment:
I'm still thinking about this. We can make this entire change, fix the bug, but still keep the API for Plugin Get the same. Providing a `found` helper value.

Internally, I think it makes sense, since as you mentioned, we are using the dynamic client, but for the plugins, does it make sense to keep the `found` value? What do you think?

Reply:
An edge case where the object is nil while found is true is if there is a layer for validating keys. In other words, a found helper can be used to communicate to plugin authors that a given key will always return a nil object. Opened #655 to implement if we reach a consensus on this.